### PR TITLE
add Daniela profile

### PR DESCRIPTION
**Pull request by SoyAndrea**

### CODE REVIEW

**Review by cosimameyer — APPROVED** [review]

**Comment by drmowinckels** [comment]
hey see #175 for information about the breaking of the build. THe automessage you can ignore, but we should make sure we have a preview built before merging.

**Comment by cosimameyer** [comment]
@drmowinckels just a quick checking in - are we actively waiting for feedback from the translation team or can we merge the branch (and close the PR)? :)

**Comment by drmowinckels** [comment]
we dont have a translation team, yet, so I think we better merge and fix later if needed :)

**Pull request merged**
